redhat/python-exabgp.spec.git

```diff
@@ -1,20 +1,29 @@
 %{!?__python2:        %global __python2 /usr/bin/python2}
 %{!?python2_sitelib:  %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%define version %(echo "$(python setup.py current).git$(git rev-parse --short HEAD)")
```

Comment:
That would work in 3.4 but would not be good for master tho ... master does not have any recent 3.4.xx tag and code. The code forked at 3.4.10 and all the fix were forward ported.